dracut/vmklive/services.sh

```diff
@@ -22,8 +22,8 @@ done
 for f in $SERVICEDIR/*; do
     _service=${f##*/}
     case "${_service}" in
-        agetty-console|agetty-generic|agetty-serial|agetty-tty[SAU]*|agetty-hv*|sulogin|dhcpcd-*|iptables|ip6tables|wpa_supplicant|pulseaudio|lvmetad|dmeventd|mdadm) ;; # ignored
-        dhcpcd) [ -n "$dhcpcd" ] && ln -sf ${f##$NEWROOT} $NEWROOT/etc/runit/runsvdir/default/;;
+        agetty-console|agetty-generic|agetty-serial|agetty-tty[SAU]*|agetty-hv*|sulogin|dhcpcd-*|iptables|ip6tables|pulseaudio|lvmetad|dmeventd|mdadm) ;; # ignored
+        dhcpcd|wpa_supplicant) [ -n "$dhcpcd" ] && ln -sf ${f##$NEWROOT} $NEWROOT/etc/runit/runsvdir/default/;;
```

Comment:
This change should be dropped and replaced by adding wpa_supplicant to $SERVICES in build-x86-images.sh.in

Reply:
as cameron said they're unable to make this change soon, I've gone ahead and done it

Reply:
I think the variants that use NetworkManager should also have wpa_supplicant enabled.

Reply:
void-docs says to disable wpa_supplicant when using networkmanager